package.json

```diff
@@ -108,7 +108,7 @@
     "stylelint-prettier": "^5.0.2",
     "typescript": "^4.9.4",
     "vite": "^6.3.4",
-    "vitest": "^3.0.7"
+    "vitest": "^3.1.1"
```

Comment:
Needed to upgrade vitest to sync the documenter test with its code-view version: https://github.com/cloudscape-design/code-view/blob/main/src/__tests__/documenter.test.ts